src/Autosuggest.js

```diff
@@ -647,10 +647,9 @@ export default class Autosuggest extends Component {
                 newValue,
                 keyCode === 40 ? 'down' : 'up'
               );
+              event.preventDefault(); // Prevents the cursor from moving.
             }
 
-            event.preventDefault(); // Prevents the cursor from moving
```

Comment:
I think there's a missed case above here for arrow up/down, when `isCollapsed && shouldRenderSuggestions(value)`. We need a `preventDefault()` in there too: if you press up/down and thereby open the suggestions menu, it shouldn't move the cursor like normal.

The case where preventDefault shouldn't be called is just when `isCollapsed && !shouldRenderSuggestions(value)`, i.e. when autosuggest does nothing with the event.

Reply:
Good catch.

Reply:
Done.